dfastmi/batch/core.py

```diff
@@ -216,46 +215,52 @@ def _report_analysis_settings_header(report: TextIO):
 def _report_basic_analysis_configuration(
     branch: Branch,
     reach: Reach,
-    q_threshold: float,
-    ucrit: float,
-    slength: float,
+    qlocation: str,
+    initialized_config: AConfigurationInitializerBase,
     report: TextIO,
 ):
     settings = {
         "branch": branch.name,
         "reach": reach.name,
-        "q_threshold": q_threshold,
-        "u_critical": ucrit,
-        "slength": int(slength),
+        "location": qlocation,
+        "q_threshold": initialized_config.q_threshold,
+        "slength": int(initialized_config.slength),
     }
     ApplicationSettingsHelper.log_text("analysis_settings", file=report, dict=settings)
 
 
-def _report_discharge_branch_location(
-    qlocation: str,
+def _report_critical_velocity(
+    ucrit: float,
     report: TextIO,
 ):
     settings = {
-        "location": qlocation,
+        "u_critical": ucrit,
     }
     ApplicationSettingsHelper.log_text(
-        "analysis_settings_discharge_location", file=report, dict=settings
+        "analysis_settings_critical_velocity", file=report, dict=settings
     )
 
 
 def _report_analysis_conditions_header(report: TextIO):
     ApplicationSettingsHelper.log_text("analysis_settings_conditions_header", report)
 
 
-def _report_used_file_names(config: ConfigParser, report: TextIO):
+def _report_used_file_names(config: ConfigParser, q_threshold: float, report: TextIO):
     for section in config:
-        if section[0].lower() == "c":
-            discharge_file_name = config.get(section, "Discharge", fallback="")
-            reference_file_name = _get_file_name(config, section, "Reference")
-            measure_file_name = _get_file_name(config, section, "WithMeasure")
-            _report_analysis_conditions_values(
-                discharge_file_name, reference_file_name, measure_file_name, report
-            )
+        if section[0].lower() == "c" or section[0].lower() == "q":
+            discharge_str = config.get(section, "Discharge", fallback="")
+            discharge = float(discharge_str)
+            condition = "{:7.1f} m3/s".format(discharge)
+            if discharge <= q_threshold:
```

Comment:
You can update this if else to make it more readable, e.g.: (I took the main version for the example, it has some more changes than suggested here)

```
        condition = "{:7.1f} m3/s".format(q)
        if q <= initialized_config.q_threshold:
            reference_file_name = _get_file_name(files[0])
            measure_file_name = _get_file_name(files[1])
            comment = ""
        elif key in filenames:
            reference_file_name = "---"
            measure_file_name = "---"
            comment = (measure not active)
        else:
            reference_file_name = "xxx"
            measure_file_name = "xxx"
            comment = (not specified)

        _report_analysis_conditions_values(
            condition, reference_file_name, measure_file_name, comment, report
        )
```

In this example I assumed the _report_analysis_conditions_values is also updated to include a comment, instead of sneaking in a comment in the measure_file_name.

In the main the _report_used_file_names method contains 2 big if elif else conditions, this could be split up in two seperate methods.

Reply:
Resolved in PR #102, comment is now part of the message, instead of added onto the measure files.

Reply:
Discussed with Bert.